load_data.py

```diff
@@ -34,19 +36,19 @@ def load_cifar(file_name, dtype, file_path='./data/cifar-10-batches-py/'):
     return data_dict
 
 
-def preprocess_cifar_data(cifar_data, dtype):
+def preprocess_cifar_data(cifar_data, cifar_labels, dtype):
     mean = np.mean(cifar_data, axis=0)
     std = np.std(cifar_data, axis=0)
     batch_size = cifar_data.shape[0]
 
-    cifar_data = (cifar_data - mean) / (std - np.finfo(dtype).eps)
+    # cifar_data = (cifar_data - mean) / (std - np.finfo(dtype).eps)
 
-    R = cifar_data[:, 0:1024]
-    G = cifar_data[:, 1024:2048]
-    B = cifar_data[:, 2048:]
+    R = cifar_data[:, 0:1024] / 255.
```

Comment:
not super pertinent to this pr, but i feel like we should standardize image preprocessing

Reply:
Depends on what you mean with standardizing. All Images of all data sets should be preprocessed the same way, I totally agree. We only need to do this consistently. Which normalization we use is less important I find.

Reply:
yes, just that it should be the same across the datasets.